test/suite/run.py

```diff
@@ -576,7 +576,7 @@ def isempty(s):
         for arg in testargs:
             testsFromArg(tests, loader, arg, scenario)
 
-    tests = hookmgr.filter_tests(tests)
+    tests = hookmgr.register_skipped_tests(tests)
```

Comment:
Since none of the `register_skip_tests` functions returns a modified list anymore, perhaps we can have it return nothing and just:
```
    hookmgr.register_skipped_tests(tests)
```
here?

Reply:
Sound to to me! Updated in [17b39ee](https://github.com/wiredtiger/wiredtiger/pull/9764/commits/17b39ee2b77ba4b8b4ac33c1721796644b0c6342)